pilot/pkg/networking/util/util.go

```diff
@@ -100,6 +100,10 @@ const (
 
 	// Well-known header names
 	AltSvcHeader = "alt-svc"
+
+	// Well-known metadata exchange EnvoyFilter config name
+	MXName110 = "metadata-exchange-1.10"
+	MXName111 = "metadata-exchange-1.11"
```

Comment:
Add a comment to remove them in 1.14

Reply:
Sure, 1.14 should be good enough. We only support skip upgrade from n-2.